core/modules/css/Facade.cc

```diff
@@ -438,17 +445,20 @@ Facade::_tableIsSubChunked(string const& dbName,
 }
 
 boost::shared_ptr<Facade>
-FacadeFactory::createMemFacade(string const& mapPath) {
+FacadeFactory::createMemFacade(string const& mapPath,
+                               std::string const& emptyChunkPath) {
     std::ifstream f(mapPath.c_str());
     if(f.fail()) {
         throw ConnError();
     }
-    return FacadeFactory::createMemFacade(f);
+    return FacadeFactory::createMemFacade(f, emptyChunkPath);
 }
 
 boost::shared_ptr<Facade>
-FacadeFactory::createMemFacade(std::istream& mapStream) {
+FacadeFactory::createMemFacade(std::istream& mapStream,
+                               std::string const& emptyChunkPath) {
     boost::shared_ptr<css::Facade> cssFPtr(new css::Facade(mapStream));
+    cssFPtr->_emptyChunks.reset(new EmptyChunks(emptyChunkPath));
```

Comment:
I'll change it to that way. It doesn't matter much to me, so whatever looks better to the group as a whole.